packages/server/src/fhircast/routes.ts

```diff
@@ -110,6 +110,13 @@ protectedRoutes.post(
     } else if (event['hub.event'].endsWith('-close')) {
       // We always close the current context, even if the event is not for the original resource... There isn't any mention of checking to see it's the right resource, so it seems it may be assumed to be always valid to do any arbitrary close as long as there is an existing context...
       await getRedis().del(`::fhircast::${req.params.topic}::latest::`);
+    } else if (event['hub.event'] === 'diagnosticreport-update') {
+      // See: https://build.fhir.org/ig/HL7/fhircast-docs/3-6-3-DiagnosticReport-update.html#:~:text=The%20Hub%20SHALL,the%20new%20updates.
+      event['context.priorVersionId'] = event['context.versionId'];
+      event['context.versionId'] = generateId();
+      stringifiedBody = JSON.stringify(req.body);
+      // TODO: Make sure this is actually supposed to be stored / overwrite open context? (ambiguous from docs, see: https://build.fhir.org/ig/HL7/fhircast-docs/2-9-GetCurrentContext.html)
+      await getRedis().set(`::fhircast::${req.params.topic}::latest::`, stringifiedBody);
```

Comment:
Out of curiosity, why double colons?   And why the double colon prefix?

In the rest of the server, we use single slash `/`, mostly because I didn't know any better when we started.

If I were to do it from scratch, I'd use single colon `:`, because a lot of Redis tools use that as a standard delimiter, which can help with visualization and debugging.

As far as I know, there is no actual behavior differences, Redis treats them all as opaque strings.  So just curious.

Reply:
I guess the idea was to kind of make it clear this is an internal key when looking through Redis, but maybe that's unnecessary. If it doesn't sit well I can change it